src/Fulma/Common.fs

```diff
@@ -423,141 +292,54 @@ module Modifier =
             match opt with
-            | BackgroundColor color -> { result with BackgroundColor = color |> ofBackground |> Some }
-            | TextColor color -> { result with TextColor = color |> ofText |> Some }
-            | TextWeight textWeight -> { result with TextWeight = textWeight |> TextWeight.ofOption |> Some }
-            | TextSize (screen, size) -> { result with TextSize = result.TextSize + " " + TextSize.generic screen size }
-            | TextSizeOnly (screen, size) -> { result with TextSize = result.TextSize + " " + TextSize.only screen size }
-            | TextAlignment (screen, size) -> { result with TextAlignment = result.TextAlignment + " " + TextAlignment.generic screen size }
-            | TextAlignmentOnly (screen, size) -> { result with TextAlignment = result.TextAlignment + " " + TextAlignment.only screen size }
-            | TextTransform transform -> { result with TextTransform = result.TextTransform + " " + TextTransform.Option.toClass transform }
-            | Display (screen, display) ->
-                let cls = Display.toDisplayClass screen display
-                { result with Display = result.Display + " " + cls }
-            | DisplayOnly (screen, display) ->
-                let cls = Display.toDisplayOnlyClass screen display
-                { result with DisplayOnly = result.DisplayOnly + " " + cls }
-            | IsClearfix -> { result with IsClearfix = Classes.Helpers.IsClearfix |> Some }
-            | IsPulledLeft -> { result with IsPulledLeft = Classes.Helpers.IsPulledLeft |> Some }
-            | IsPulledRight -> { result with IsPulledRight = Classes.Helpers.IsPulledRight |> Some }
-            | IsMarginless -> { result with IsMarginless = Classes.Helpers.IsMarginless |> Some }
-            | IsPaddingless -> { result with IsPaddingless = Classes.Helpers.IsPaddingless |> Some }
-            | IsOverlay -> { result with IsOverlay = Classes.Helpers.IsOverlay |> Some }
-            | IsClipped -> { result with IsClipped = Classes.Helpers.IsClipped |> Some }
-            | IsRadiusless -> { result with IsRadiusless = Classes.Helpers.IsRadiusless |> Some }
-            | IsShadowless -> { result with IsShadowless = Classes.Helpers.IsShadowless |> Some }
-            | IsUnselectable -> { result with IsUnselectable = Classes.Helpers.IsUnselectable |> Some }
-            | IsInvisible (screen, true) -> { result with IsInvisible = result.IsInvisible + " " + ofInvisible screen }
-            | IsInvisible (_, false) -> result
-            | IsHidden (screen, true) -> { result with IsHidden = result.IsHidden + " " + ofHidden screen }
-            | IsHidden (_, false) -> result
-            | IsInvisibleOnly (screen, true) -> { result with IsInvisibleOnly = result.IsInvisibleOnly + " " + ofInvisible screen }
-            | IsInvisibleOnly (_, false) -> result
-            | IsHiddenOnly (screen, true) -> { result with IsHiddenOnly = result.IsHiddenOnly + " " + ofHidden screen }
```

Comment:
This is using `ofHidden screen` (and two lines above `ofInvisible screen`). Is it an error? I've changed them to `ofHiddenOnly` and `ofInvisibleOnly` respectively.

Reply:
Yes, I guess it was an error I will need to check when reviewing the whole PR.